phoenix-core/src/it/java/org/apache/phoenix/rpc/UpdateCacheIT.java

```diff
@@ -104,10 +111,30 @@ public void testUpdateCacheForNeverUpdatedTable() throws Exception {
         String tableName = generateUniqueName();
         String fullTableName = INDEX_DATA_SCHEMA + QueryConstants.NAME_SEPARATOR + tableName;
         Properties props = PropertiesUtil.deepCopy(TEST_PROPERTIES);
+        AtomicBoolean isSysMutexEmpty = new AtomicBoolean(true);
+        ExecutorService executorService = Executors.newFixedThreadPool(5,
+            new ThreadFactoryBuilder().setDaemon(true)
+                .setNameFormat("check-sys-mutex-count-%d").build());
+        Connection connForMutex = DriverManager.getConnection(getUrl(), props);
+        for (int i = 0; i < 5; i++) {
+            executorService.submit(new SystemMutexCaller(isSysMutexEmpty,
+                connForMutex));
+        }
+        Thread.sleep(500);
```

Comment:
Sort of warm up for threads to start executing select query :)
But yes, since we are not achieving anything major here, let me remove it if you are strong about it?